cowprotocol/accounting/rewards/mainnet/mainnet_dashboard_query_2510345.sql

```diff
@@ -0,0 +1,222 @@
+with
+
+block_range as (
+    select * from "query_3333356(start_time='{{start_time}}',end_time='{{end_time}}')"
+),
+
+results_per_tx as (
+    select * from "query_3427730(start_time='{{start_time}}',end_time='{{end_time}}',cte_name='results_per_tx')"
+
+),
+
+solver_slippage as (
+    select
+        solver_address as solver,
+        sum(eth_slippage_wei) * 1.0 / pow(10, 18) as slippage
+    from results_per_tx
+    group by solver_address
+),
+
+named_results as (
+    select * from "1541516(end_time='{{end_time}}',cte_name='named_results')"
+),
+-- end SLIPPAGE
```

Comment:
slippage ends after `solver_slippage`, right?

Reply:
removed the comments